File: core/src/com/fangelo/kotlinrpg/ui/dialog/ConfirmDialog.java
package com.fangelo.kotlinrpg.ui.dialog;

import com.fangelo.libraries.ui.Dialog;
import com.fangelo.libraries.ui.DialogResult;

public class ConfirmDialog extends Dialog {

	public ConfirmDialog(String title, String text) {
		super(title);

		text(text);

		button("Yes", DialogResult.Yes);
		button("No", DialogResult.No);
	}
}

File: core/src/com/fangelo/kotlinrpg/ui/dialog/ConfirmDialog.kt
package com.fangelo.kotlinrpg.ui.dialog

import com.fangelo.libraries.ui.Dialog
import com.fangelo.libraries.ui.DialogResult

class ConfirmDialog(title: String, text: String) : Dialog(title) {

    init {

        text(text)

        button("Yes", DialogResult.Yes)
        button("No", DialogResult.No)
    }
}
